### PR TITLE
fix: typo in CronDetector searchpath

### PR DESCRIPTION
**Pull request by boredland**
I read up code on the CronDetector - found a typo and added a line more to the docs to make it easier for others to understand.

### CODE REVIEW

File: docs/docs/deploying.md [hunk 1]
```diff
@@ -6,27 +6,28 @@ Deploying Quirrel is straight-forward.
...
+Make sure to have `QUIRREL_API_URL`, a `QUIRREL_TOKEN` and the `QUIRREL_BASE_URL` set when executing `quirrel ci`.
 
 ```json
 "scripts": { "vercel-build": "npm run build && quirrel ci" }
 ```
 
 :::
 
-:::note VERCEL_URL
+:::note VERCEL*URL
```
Comment (boredland) [inline docs/docs/deploying.md:28]: this is so confusing. in my ide:

image

**Review by Skn0tt — CHANGES_REQUESTED** [review]
good catch with the typo + lovely addition to docs, thank you :)

it looks like there are some things that your IDE changed with the markdown, please revert tose and reduce the changes to the typo + docs.

**Comment by boredland** [comment]
it is confusing. the changes, it seems, are applied with an on-commit hook, not by vscode. however: I reverted them. how about that line-break it removed in the console statement. to me, now it looks less weird. but perhaps there is a reason you had it as it was?

**Review by Skn0tt — APPROVED** [review]
LGTM! weird that there was a commit hook 🤔

**Pull request merged**